circle.yml

```diff
@@ -0,0 +1,14 @@
+dependencies:
+  override:
+    - pip install awscli==1.11.139
+
+test:
+  override:
+    - test 0 = 0
+
+deployment:
```

Comment:
Cool stuff. Let's push any non-master branch to staging as well please, so we have a way to see changes together with the styling and share with other people to review